src/parser/csql_grammar.y

```diff
@@ -20765,8 +20799,36 @@ class_comment_spec
 			    PT_NODE_PRINT_VALUE_TO_TEXT (this_parser, node);
 			  }
 
+			PT_NODE *alter_node = parser_get_alter_node();
+
+			if (alter_node)
+			  {
+				alter_node->info.alter.code = PT_CHANGE_TABLE_COMMENT;
```

Comment:
When using class_comment_spec, info.alter.code is not always PT_CHANGE_TABLE_COMMENT, so do not set the info.alter.code value to PT_CHANGE_TABLE_COMMENT in class_comment_spec.
So, i will change the info.alter.code value to be set to PT_CHANGE_TABLE_COMMENT only if it is not PT_CHANGE_COLUMN_COMMENT at the original location.